docs/topics/plugin-icon-legend.adoc

```diff
@@ -5,17 +5,17 @@
 [id='plugin-icon-legend_{context}']
 = About {ProductShortName} issue icons
 
-In the Issue Explorer and file editors, {ProductShortName} issues use an icon to indicate their severity level and status. The following table describes the meaning of the various icons.
+{ProductShortName} displays icons in *Issue Explorer* and file editors to indicate the severity and status of issues related to migration. The table below describes the meaning of these icons.
 
 .{ProductShortName} issue icons
 [cols="20%a,80%",options="header",]
 |====
```

Comment:
Organizational suggestion: Move the non-quick fix icons to the section on reviewing issues, to step 2. Then delete this module and its parent section.